src/inputs/forms/json-schema-form/PJsonSchemaForm.vue

```diff
@@ -55,16 +55,26 @@
                                        :disabled="schemaProperty.disabled"
                                        @update:selected="handleUpdateFormValue(schemaProperty, ...arguments)"
                     />
-                    <p-text-input v-else
-                                  :value="rawFormData[schemaProperty.propertyName]"
-                                  :type="schemaProperty.inputType"
-                                  :invalid="invalid"
-                                  :placeholder="schemaProperty.inputPlaceholder"
-                                  :masking-mode="schemaProperty.inputType === 'password'"
-                                  :autocomplete="false"
-                                  :disabled="schemaProperty.disabled"
-                                  @update:value="handleUpdateFormValue(schemaProperty, ...arguments)"
-                    />
+                    <template v-else>
+                        <p-text-input v-if="schemaProperty.type === 'array'"
+                                      :selected="rawFormData[schemaProperty.propertyName]"
+                                      :type="schemaProperty.inputType"
+                                      :invalid="invalid"
+                                      :placeholder="schemaProperty.inputPlaceholder"
+                                      :disabled="schemaProperty.disabled"
+                                      multi-input
+                                      @update:selected="handleUpdateFormValue(schemaProperty, ...arguments)"
+                        />
+                        <p-text-input v-else
+                                      :value="rawFormData[schemaProperty.propertyName]"
+                                      :type="schemaProperty.inputType"
+                                      :invalid="invalid"
+                                      :placeholder="schemaProperty.inputPlaceholder"
+                                      :masking-mode="schemaProperty.inputType === 'password'"
+                                      :disabled="schemaProperty.disabled"
+                                      @update:value="handleUpdateFormValue(schemaProperty, ...arguments)"
+                        />
+                    </template>
```

Comment:
```suggestion
     <p-text-input v-else
        :value="schemaProperty.multiInputMode ? undefined : rawFormData[schemaProperty.propertyName]"
        :selected="schemaProperty.multiInputMode ? rawFormData[schemaProperty.propertyName] : undefined"
        :type="schemaProperty.inputType"
        :invalid="invalid"
        :placeholder="schemaProperty.inputPlaceholder"
        :masking-mode="schemaProperty.inputType === 'password'"
        :autocomplete="false"
        :disabled="schemaProperty.disabled"
        :multi-input="schemaProperty.multiInputMode"
        @update:value="handleUpdateFormValue(schemaProperty, ...arguments)"
/>              
```

How about adding a new value "multiInputMode" to the internal schema property to minimize the branching statement?

---

내부 스키마 프로퍼티에 "multiInputMode" 라는 새로운 값을 추가하여 분기문을 최소화하는 것은 어떨까요?

Reply:
Good Ideaaaa